### ISSUE TITLE
Online Diagnostic and GPS

### ISSUE THREAD

**Issue by aytekno** [x893/CarTracker#1]
Hello,
We want to use your product for online diagnostic and tracking. We do not want the processes online, just want to record the data on a SD card . We opened the pcb files and we could not see micro sd card slot. In your previous design there was. What do you prefer? Have you ever try this card for CanBus?
Thank you in advance.

**Comment by x893** [comment]
Hello, SD card not need for my application. But you can add SD card to board or redesign for your box. My board (both variants) contains CAN adapter but also you can add modern CAN bus variants. So you can change as you need. You can contact with me via skype - see in top left corner on my site.

**Comment by toxatt** [comment]
Hello. Have you realization collecting data to flash memory when have no GSM signal?

**Comment by x893** [comment]
Yes, many years ago. As ring buffer but for microtracker (MX25 flash chip). See on site my site for device. Algorithm very simple - nothing interesting.
